packages/next/build/webpack/config/blocks/css/loaders/utils/SequentialIDGenerator.ts

```diff
@@ -0,0 +1,39 @@
+// Utility class used to help us minify CSS classnames
+// down to two characters each for production builds.
+//
+// Original: https://github.com/styletron/styletron/blob/master/packages/styletron-engine-atomic/src/sequential-id-generator.js
+// Modifications:
+//   - single-character id generation
+//   - use base62 instead of base36
```

Comment:
I'd give some kind of justification for this as well, perhaps some example files from the output of an example site with the size of different approaches applied.

This could let you further investigate opportunities in the future with a stable resource to test against.

Also, it would make for a nice test to ensure changes don't break this minification approach in the future.... sample resource A minifies to less than X bytes without compression, Y bytes with gzip, and Z bytes with brotli.